SparkPost/2web2ui
# SA-1256 - Remove Stop Inbox Placement Test

### What Changed
 - Only UI change here is the removal of the "Stop Test" button on the inbox placement on the details page of an inbox placement test

### How To Test
 - Start an inbox placement test 
 - Ensure the "Stop Test" button is not in the top right corner of the details page (You can verify that it _would_ be there if you check staging/prod (whichever your local environment is pointing to)
